package.json

```diff
@@ -17,14 +17,14 @@
   },
   "repository": {
     "type": "git",
-    "url": "https://github.com/origin/origin-web-catalogs.git"
+    "url": "https://github.com/origin/origin-web-catalog.git"
   },
-  "author": "Jeff Phillips, David Taylor",
+  "author": "Red Hat, Inc",
   "license": "MIT",
```

Comment:
I'm not seeing a LICENSE file at all, can we go ahead and fix that too, grab the one in origin-web-console